src/igr-tests/igr-runner.cc

```diff
@@ -55,7 +56,8 @@ int main(int argc, char **argv)
             { "pseudo-cycle", pseudo_cycle_test }, { "before-after", before_after_test},
             { "before-after2", before_after2_test }, { "log-via-pipe", log_via_pipe_test },
             { "catlog", catlog_test }, { "offline-enable", offline_enable_test },
-            { "xdg-config", xdg_config_test }, { "cycles", cycles_test } };
+            { "xdg-config", xdg_config_test }, { "cycles", cycles_test },
+            { "arg", arg_test } };
```

Comment:
The `arg` is too generic for this test, It's confusing in the first look. I suggest to use `svc-arg` instead of just `arg` for this test.